changelog.d/metavar-pattern-generic.feat

```diff
@@ -0,0 +1 @@
+Allow metavariable-pattern clauses that use `language: generic` to run (and potentially match) on any metavariable binding kind, not just strings.
```

Comment:
Isn't this change associated with a customer-related ticket? I would expect the ticker number to be used for the changelog entry instead of `metavar-pattern-generic.feat` and referenced in the PR comment as well.

Reply:
We could also add more detail in this changelog entry (like an example use case), so that tech-writers can highlight it appropriately in the release notes. cc @adamkvitek

Reply:
Well, there are going to be at least 3 PRs associated with that ticket and they can't all use the same changelog file. Doing it this way seems to make the most sense to me.

Reply:
I've added an example to the changelog entry.